entries/QUnit.config.xml

```diff
@@ -35,6 +35,13 @@
 				<p>Prior to version 1.16.0, this property was called <code>module</code>.</p>
 			</desc>
 		</property>
+		<property name="random" type="Number" default="undefined">
+			<desc>
+				This property tells QUnit to run tests in a seeded-random order. The value provided will be used as the seed in a pseudo-random number generator to ensure that results are reproducible.
+				<p>Randomly ordering your tests can help identify non-atomic tests which either depend on a previous test or are leaking state to following tests. This is particularly beneficial in a development CI or post-commit process.</p>
+				<p>If <code>random</code> is specified in the page's url parameters, but no seed value is specified, then QUnit will generate a random seed to use for the test run. You can access this seed from this property in order to reproduce any potential test failures.</p>
+			</desc>
```

Comment:
After updating the feature to use [xorshift](http://excamera.com/sphinx/article-xorshift.html) as the randomization strategy, I think it makes more sense to use `seed`. The "default" value can be `0`, since that results in no randomization (would require a slight tweak to the other PR though).